app/main.py

```diff
@@ -1 +1,42 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str, appetite: int, is_hungry=True):
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
+
+
+class Cat(Animal):
+    def __init__(self, name: str, is_hungry=True):
+        super().__init__(name, appetite=3)
+        self.name = name
+        self.is_hungry = is_hungry
```

Comment:
Redundant assignments: The `name` and `is_hungry` attributes are already set in the `Animal` class constructor via `super().__init__()`. These assignments can be removed to avoid redundancy.